web/src/components/TopNavigation.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react'
+import { Menu } from 'antd'
+import { Link } from 'react-router-dom'
+import { QuestionCircleOutlined, UserOutlined } from '@ant-design/icons'
+import { useMeQuery } from '../types/api'
+import { useLocalStorageState } from '@umijs/hooks'
+
+export function TopNavigation(): ReactElement {
+  const [code] = useLocalStorageState('code', '')
```

Comment:
nit:
1. 'code' -> use enum / const
2. wrap entire useLocalStorageState('code', '') with another hook `useLoginCode`. To hide the implementation (which is one line of code, but ..) and also it will hide string `code`